qcodes/tests/drivers/test_keysight_34934a.py

```diff
@@ -0,0 +1,168 @@
+# pylint: disable=redefined-outer-name
```

Comment:
No action items here, but I really don't understand why this is needed here and not in any other test file.

Reply:
it might have to do with `.pylintrc` and Codacy... i think with time we'll figure this out :)